lib/inspec/resources/service.rb

```diff
@@ -510,6 +519,37 @@ def info(service_name)
     end
   end
 
+  # @see: https://www.freebsd.org/doc/en/articles/linux-users/startup.html
+  # @see: https://www.freebsd.org/cgi/man.cgi?query=rc.conf&sektion=5
+  # @see: https://www.freebsd.org/cgi/man.cgi?query=rc&apropos=0&sektion=8&manpath=FreeBSD+10.0-RELEASE&arch=default&format=html
+  class FreeBSD10Init < ServiceManager
```

Comment:
I have no opinion at all on whether it is better to name the new one `BSDInit` and rename the old one to `BSDInitOld` or `FreeBSD6Init`, or whatever.